docs/editor_guide/segments_dashboard.rst

```diff
@@ -0,0 +1,84 @@
+The segments dashboard
+======================
+
+Wagxperience comes with two different views for it's segment dashboard. A "list
+view" and a "dashboard view". Where the dashboard view attempts to show all
+relevant information and statistics in a visually pleasing manner, the list view
+is more fitted for sites using large amounts of segments, as it may be
+considered more clear in these cases.
+
+
+Switching between views
+-----------------------
+
+By default, Wagxperience's "dashboard view" is active on the segment dashboard.
+If you'd like to switch between the dashboard view and list view, open the
```

Comment:
Was hem al een paar keer tegen gekomen, persoonlijk vind ik ``you would`` beter klinken dan ``you'd``